Makefile.am

```diff
@@ -213,6 +213,56 @@ clean-local:
 	-rm -rf t/cover_db/
 
 t/cover_db/: check
+COVER_OPTS= \
+	PERL5OPT="-MDevel::Cover=-db,$(abs_builddir)/cover_db,+inc,$(PWD),-select,$(PWD)/[^/]+.pm|$(PWD)/(OpenQA|consoles|backend)|$(PWD)/isotovideo,-ignore,/usr/bin/prove|t/data/|t/fake/|\.t|tools/lib/perlcritic"
+TEST_OPTS = \
+	PERL5LIB="$(PWD):$(PWD)/ppmclibs/blib/lib:$(PWD)/ppmclibs/blib/arch/auto/tinycv:$$PERL5LIB"
+
+TEST_FILES = \
+	00-compile-check-all.t \
+	01-test_needle.t \
+	02-test_ocr.t \
+	03-testapi.t \
+	04-check_vars_docu.t \
+	05-pod.t \
+	06-pod-coverage.t \
+	07-commands.t \
+	08-autotest.t \
+	09-lockapi.t \
+	10-terminal.t \
+	10-test-image-conversion-benchmark.t \
+	11-image-ppm.t \
+	12-bmwqemu.t \
+	13-osutils.t \
+	14-isotovideo.t \
+	15-logging.t \
+	16-send_with_fd.t \
+	17-basetest.t \
+	18-qemu.t \
+	18-backend-qemu.t \
+	18-qemu-options.t \
+	19-isotovideo-command-processing.t \
+	20-openqa-benchmark-stopwatch-utils.t \
+	21-needle-downloader.t \
+	22-svirt.t \
+	23-baseclass.t \
+	24-myjsonrpc.t \
+	24-myjsonrpc-debug.t \
+	25-spvm.t \
+	99-full-stack.t
+
+test:
+	(cd t && $(TEST_OPTS) prove $(TEST_FILES) )
+
+test-cover:
+	(cd t && $(TEST_OPTS) $(COVER_OPTS) prove $(TEST_FILES) )
+	cover -summary cover_db
+
+test-cover-summary:
+	cover -summary cover_db
+
+test-cover-codecov: cover_db/
+	cover -report codecov cover_db
 
 .PHONY: coverage-test
 coverage-test: t/cover_db/
```

Comment:
now we have rules for `cover_db/` as well as `t/cover_db/`. Maybe this is now obsolete?

Reply:
I thought I'd keep it for comparison as long as I'm not sure the new target is working well.